README.md

```diff
@@ -91,7 +91,7 @@ RAILS_ENV=production bin/validate-cocina -p 8
 
 ## Running Reports in DSA
 
-Custom reports stored in dor-services-app can be run similarly to validation testing described above.
+Custom reports stored in dor-services-app can be run similarly to validation testing described above.  The reports are stored in the `app/reports` directory in dor-services-app.  Examine some of the reports in that folder to see the pattern of how they are structured and run.  Since they need to access to the SDR datastore to run, you can run them on the `sdr-infra` box, which has read-only access to each environment (qa/stage/prod).
```

Comment:
Should all of this be moved to the DSA README?

Reply:
https://github.com/sul-dlss/dor-services-app/pull/5561